qemu/tests/drive_mirror.py

```diff
@@ -131,8 +152,6 @@ def wait_for_steady(self):
         """
         params = self.parser_test_args()
         timeout = params.get("wait_timeout")
-        if self.vm.monitor.protocol == "qmp":
```

Comment:
With this 'clear_event("BLOCK_JOB_READY")', it will only lead to time out error when looping block job status, even when block job actually got ready, and I could get correct detecting result without this clearing.